### ISSUE TITLE
MS Sans Serif bold and italic rendering

### ISSUE THREAD

**Issue by gayfieri** [grassmunk/Chicago95#267]
i've seen people have some issues with fonts before but i haven't seen any screenshots that match what it looks like on my end. here's what they look like for me after following the installation guide

![Screenshot_2022-10-12_13-22-36](https://user-images.githubusercontent.com/94586008/195468403-82685e97-4351-49ba-a062-375cd84063cd.png)

![Screenshot_2022-10-12_13-24-39](https://user-images.githubusercontent.com/94586008/195468418-f0799ff3-cb5a-4545-b50b-28777c841a6c.png)

![Screenshot_2022-10-12_13-17-20](https://user-images.githubusercontent.com/94586008/195468441-68c10a56-1aa1-48a0-acb4-06c938d43525.png)

![Screenshot_2022-10-12_13-29-52](https://user-images.githubusercontent.com/94586008/195468455-2c102052-9d3d-4bff-93cf-ed9725db0092.png)

![Screenshot_2022-10-12_13-28-21](https://user-images.githubusercontent.com/94586008/195468463-7cf580ff-e2fa-4493-ab35-4b745bcc85a0.png)

sorry if this is a duplicate

**Comment by 1chan-level-journalist** [comment]
I've been making a character-for-character trace of MS Sans Serif in FontForge, including Bold version of it. I called it "MS Sans Serif Raster". 
Here's how it looks:
![screenshot](https://user-images.githubusercontent.com/16210758/209997887-ecec9b90-cae4-407a-afbd-8d0ed5ad2f96.png)
Try it out!
[MSSansSerifRaster.zip](https://github.com/grassmunk/Chicago95/files/10321063/MSSansSerifRaster.zip)

**Comment by 1chan-level-journalist** [comment]
Yeah, the synthetic emboldening didn't work right for me either - it didn't look strangely pixelated, but font kerning didn't work, resulting in letters being squished together too close. That's why I ended up making a bold version of the font completely from scratch (just like the cronyx/Helvetica font).
